src/Softeq.NetKit.Chat.Tests/Integration/RepositoryTests/MemberRepositoryTests.cs

```diff
@@ -369,5 +370,60 @@ public async Task GetAllMembersByChannelIdAsync_ShouldReturnAllMembersInChannel(
             // Assert
             membersFromChannel.Should().BeEquivalentTo(expectedMembers);
         }
+
+        [Fact]
```

Comment:
Did you run unit tests? Is everything works as expected? We need to get latest develop and rerun the unit tests.